### PR TITLE
Updated cirq.ionq to cirq_ionq throughout

### PR DESCRIPTION
**Pull request by dwiddows**
Fixes out-of-date documentation. The correct import throughout is now `import cirq_ionq` rather than `import cirq.ionq`.

### CODE REVIEW

File: docs/ionq/circuits.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ these gates are one or two qubit gates which support the `unitary` protocol
 for the gate) there is support for compiling these into API supported gates.
 This conversion may not be optimal, but it does produce a valid API circuit.
 
-This support is given by the `cirq.ionq.IonQAPIDevice` and its
+This support is given by the `cirq_ionq.IonQAPIDevice` and its
 `decompose_operation` method.  On way to use this is to pass the device
```
Comment (tanujkhattar) [inline docs/ionq/circuits.md:100]: We also need to update this documentation to use the new transformers for compilation (see https://github.com/quantumlib/Cirq/pull/5479). The decompose_operation method as on the device has been deprecated and removed. 

Can you please file a new issue and assign to yourself / someone from IonQ ?

**Review by tanujkhattar — APPROVED** [review]

**Review by tanujkhattar — CHANGES_REQUESTED** [review]
@dwiddows Can you please sign the CLA? https://cla.developers.google.com/clas

**Comment by dwiddows** [comment]
> 

Done the CLA registration part.

I filed an issue at https://github.com/quantumlib/Cirq/issues/5497, can't assign it but there's a comment on the ticket requesting it to be assigned to @Cynocracy .

File: docs/ionq/circuits.md (hunk 1 above)
Comment (dwiddows) [reply docs/ionq/circuits.md:100]: Started - I filed an issue at https://github.com/quantumlib/Cirq/issues/5497, can't assign it yet but there's a comment on the ticket requesting it to be assigned to @Cynocracy .

**Review by tanujkhattar — APPROVED** [review]

**Comment by dwiddows** [comment]
Thanks very much for reviewing and merging this, @tanujkhattar.

**Pull request merged**